src/Resources/skeleton/forgottenPassword/twig_check_email.tpl.php

```diff
@@ -0,0 +1,6 @@
+<?= $helper->getHeadPrintCode('Check your e-mail address'); ?>
+
+{% block body %}
+    <p>An email has been sent. It contains a link you must click to reset your password. This link will expires in {{ tokenLifetime }} hours.</p>
+    <p>If you don't get an email please check your spam folder or try again.</p>
```

Comment:
And we can wrap "try again" with a link to `app_forgotten_password` route for convenience

Reply:
Good idea!